content/patterns/disclosure/disclosure-pattern.html

```diff
@@ -14,47 +14,49 @@
-          It has two elements: a disclosure <a href="../button/button-pattern.html">button</a> and a section of content whose visibility is controlled by the button.
-          When the controlled content is hidden, the button is often styled as a typical push button with a right-pointing arrow or triangle to hint that activating the button will display additional content.
-          When the content is visible, the arrow or triangle typically points down.
+          ディスクロージャーは、コンテンツを折りたたむ（非表示）または展開（表示）することができるウィジェットです。
+          ディスクロージャーには次の 2 つの要素があります：ディスクロージャー <a href="../button/button-pattern.html">ボタン</a>と、ボタンによって可視性が制御されるコンテンツのセクションです。
+          制御されたコンテンツが非表示のとき、ボタンはしばしば、追加のコンテンツが表示されることを示唆する右向きの矢印または三角形が付いた典型的なプッシュボタンとしてスタイリングされます。
+          コンテンツが表示されているとき、矢印または三角形は通常、下を指します。
         </p>
       </section>
 
       <section id="examples">
-        <h2>Examples</h2>
+        <img alt="" src="../../images/pattern-disclosure.svg">
+        <h2>例</h2>
         <ul>
-          <li><a href="examples/disclosure-image-description.html">Disclosure (Show/Hide) of Image Description</a></li>
-          <li><a href="examples/disclosure-faq.html">Disclosure (Show/Hide) of Answers to Frequently Asked Questions</a></li>
-          <li><a href="examples/disclosure-navigation.html">Disclosure (Show/Hide) Navigation Menu</a></li>
-          <li><a href="examples/disclosure-navigation-hybrid.html">Disclosure (Show/Hide) Navigation Menu with Top-Level Links</a></li>
+          <li><a href="examples/disclosure-image-description.html">画像説明のディスクロージャー（表示/非表示）</a></li>
+          <li><a href="examples/disclosure-faq.html">よくある質問への回答のディスクロージャー（表示/非表示）</a></li>
+          <li><a href="examples/disclosure-navigation.html">ディスクロージャー（表示/非表示）ナビゲーションメニュー</a></li>
+          <li><a href="examples/disclosure-navigation-hybrid.html">トップレベルリンク付きディスクロージャー（表示/非表示）ナビゲーションメニュー</a></li>
         </ul>
       </section>
 
       <section id="keyboard_interaction">
-        <h2>Keyboard Interaction</h2>
-        <p>When the disclosure control has focus:</p>
+        <h2>キーボードインタラクション</h2>
+        <p>ディスクロージャー制御がフォーカスを受けたとき：</p>
         <ul>
-          <li><kbd>Enter</kbd>: activates the disclosure control and toggles the visibility of the disclosure content.</li>
-          <li><kbd>Space</kbd>: activates the disclosure control and toggles the visibility of the disclosure content.</li>
+          <li><kbd>Enter</kbd>: ディスクロージャー制御をアクティブにし、ディスクロージャーコンテンツの可視性を切り替えます。</li>
+          <li><kbd>Space</kbd>: ディスクロージャー制御をアクティブにし、ディスクロージャーコンテンツの可視性を切り替えます。</li>
         </ul>
       </section>
 
       <section id="roles_states_properties">
-        <h2>WAI-ARIA Roles, States, and Properties</h2>
+        <h2>WAI-ARIA ロール、ステート、およびプロパティ</h2>
         <ul>
-          <li>The element that shows and hides the content has role <a href="#button" class="role-reference">button</a>.</li>
+          <li>コンテンツを表示および非表示にする要素はロール<a href="#button" class="role-reference">button</a>を持ちます。</li>
```

Comment:
英単語の前後に半角スペースを入れました。

```suggestion
          <li>コンテンツを表示および非表示にする要素はロール <a href="#button" class="role-reference">button</a> を持ちます。</li>
```